### DISCUSSION TITLE
Feature: Change Manufacture Name

### DISCUSSION THREAD

**Discussion by fishooo0815** [halo/LinkLiar#54] [Ideas]
The Manufacture Field in Router View is empty when using LinkLiar.

Would be nice to change this too.

**Comment by halo** [comment]
Hi!

I don't understand. Do you have a screenshot to explain what you mean?

**Reply by fishooo0815** [reply]
In my router there is a Info Page:
```
NAME     IP ADDRESSS      MANUFACTURER               MAC
Mac-01   192.168.1.54      Apple                          11:22:33:aa:bb:cc
iPhone    192.168.1.57       Apple                         66:22:33:aa:bb:cc
Dell        192.168.1.43                              33:33:33:33:bb:aa
```
In the Example the Dell does not has a Manufacture Name assigned.

Can I change this too?

But after rethinking. I guess this is generated by the router based on a database of mac adresses.

What do you think?

**Reply by halo** [reply]
> But after rethinking. I guess this is generated by the router based on a database of mac adresses.

That's exactly what happens. Your router takes the first 6 characters from the MAC address and looks up the vendor name in a list. It's a so called OUI list.

Such as this: https://gitlab.com/wireshark/wireshark/-/raw/master/manuf

There is nothing you can do on your end (in LinkLiar or your laptop) to change what the router displays.
